Include/internal/pycore_runtime.h

```diff
@@ -55,74 +55,81 @@ typedef struct _Py_DebugOffsets {
     uint64_t version;
     // Runtime state offset;
     struct _runtime_state {
-        off_t finalizing;
-        off_t interpreters_head;
+        uint64_t finalizing;
+        uint64_t interpreters_head;
```

Comment:
Because `off_t` is platform dependant (32/64 bit) and that makes capturing it in a generic way super annoying and you will need to fight implicit conversions anyway. This makes these numbers 64 bit long in all cases, which for 99% of machines (64) bit makes no difference and for the rest it's a bit bigger but more predictable